--- lib/lsb/init-functions.d/40-systemd.sav	2015-06-17 23:21:53.203426681 +0200
+++ lib/lsb/init-functions.d/40-systemd	2015-06-17 23:22:26.439207775 +0200
@@ -23,6 +23,15 @@
                 if [ "$(systemctl -p CanReload show $service 2>/dev/null)" = "CanReload=no" ] && [ "${1:-}" = "reload" ]; then
                     _use_systemctl=0
                 fi
+                # XXX makinacorpus_container_init,
+                # some service redirections are not working in a container context
+                # where systemd pid != 1
+                if grep [ -f /.dockerinit ];then
+                  _use_systemctl=0
+                fi
+                if grep -q /lxc/ /proc/1/group 2>/dev/null;then
+                  _use_systemctl=0
+                fi
                 ;;
         esac
     else
