diff --git a/files/etc/apparmor.d/usr.sbin.ntpd b/files/etc/apparmor.d/usr.sbin.ntpd
index 48e8d7a..afbff24 100644
--- a/files/etc/apparmor.d/usr.sbin.ntpd
+++ b/files/etc/apparmor.d/usr.sbin.ntpd
@@ -13,7 +13,7 @@
 
 #include <tunables/global>
 #include <tunables/ntpd>
-/usr/sbin/ntpd {
+/usr/sbin/ntpd flags=(attach_disconnected) {
   #include <abstractions/base>
   #include <abstractions/nameservice>
   #include <abstractions/user-tmp>
